src/Disks/ReadIndirectBufferFromRemoteFS.cpp

```diff
@@ -87,14 +87,17 @@ bool ReadIndirectBufferFromRemoteFS<T>::nextImpl()
 {
     /// Find first available buffer that fits to given offset.
     if (!current_buf)
+    {
         current_buf = initialize();
+        pos = working_buffer.begin();
```

Comment:
Does this line makes any real effect?

Reply:
Not sure, it may be a debugging artifact.